pkg/controllers/csi/provisioner/controller.go

```diff
@@ -225,6 +226,11 @@ func (provisioner *OneAgentProvisioner) updateAgentInstallation(ctx context.Cont
 			return nil, false, err
 		}
 		latestProcessModuleConfig.AddProxy(proxy)
+
+		if dk.NeedsActiveGate() {
+			multiCap := capability.NewMultiCapability(dk)
+			latestProcessModuleConfig.AddNoProxy(capability.BuildDNSEntryPoint(dk.Name, dk.Namespace, multiCap))
+		}
```

Comment:
so far it looks good, but what about the `applicationMonitoring without CSI-Driver` scenario?
do we support that?
if so then you have to do some magic from the init-container side

Reply:
do you mean when we have ActiveGate and without CSI ?

Reply:
no, ActiveGate + applicationMonitoring without CSIDriver (so only injection, where each pod downloads the agent on its own)

the problem there:
- the init-container does not directly know about the Dynakube
- you could try [creating the NoProxy thing directly in the ini-secret](https://github.com/Dynatrace/dynatrace-operator/blob/010623f6bdc6601856fffa71b44944a7dd2ee925/pkg/injection/namespace/initgeneration/initgeneration.go#L164-L164)

Reply:
fixed in 9985b747